src/realm.json

```diff
@@ -5,8 +5,8 @@
     "displayNameHtml": "Identity Service",
     "notBefore": 0,
     "defaultSignatureAlgorithm": "RS256",
-    "revokeRefreshToken": false,
-    "refreshTokenMaxReuse": 0,
+    "revokeRefreshToken": true,
+    "refreshTokenMaxReuse": 1,
```

Comment:
Should we document this in our realm/identity config upgrade doc (https://github.com/defenseunicorns/uds-identity-config/blob/main/docs/reference/UDS%20Core/IdAM/upgrading-versions.md)?

Sidenote I also realized our docs links on the readme are currently broken ([here](https://github.com/defenseunicorns/uds-identity-config/blob/main/README.md#customizing-uds-identity-config) and [here](https://github.com/defenseunicorns/uds-identity-config/blob/main/README.md#upgrading-identity-config)).